spec/system/provider_interface/manage_provider_user_permissions_spec.rb

```diff
@@ -3,6 +3,8 @@
 RSpec.feature 'Managing provider user permissions' do
   include DfESignInHelpers
 
+  before { FeatureFlag.deactivate(:account_and_org_settings_changes) }
```

Comment:
The hardest bit about removing the feature flag last time was figuring out what to do with specs like these.

Maybe we could put a comment next to each of the `deactivate` calls in specs explaining what the plan is. It'll be one of:
- we are removing this behaviour
- we are moving this behaviour elsewhere and here's the new spec

For the latter if there isn't a new spec yet, create a placeholder with the very first bit of the journey.

What do you think?

Reply:
Alright, I've made placeholders for each system spec that we're going to be affecting - it turns out there's quite a few which will need tweaks.

I'm not sure if there's a better naming convention than putting `placeholder` in front of all of them, but I don't think it's correct to try and come up with alternative names for each one, as that gets more confusing

Reply:
I think we have now decided not to add placeholder specs. Instead, we are going to aim to put comments on the specs we have reinstated the behaviour of elsewhere. When removing the flag it will be easy to remove such old specs.